aries-site/src/examples/components/table/TableSingleSelectExample.js

```diff
@@ -1,5 +1,23 @@
 import React from 'react';
-import { Box, DataTable, Heading, Text, ResponsiveContext } from 'grommet';
+import {
+  Box,
+  DataTable,
+  Heading,
+  Text,
+  ResponsiveContext,
+  Button,
+} from 'grommet';
+import { FormPrevious } from 'grommet-icons';
+
+const orderDeatils = [
```

Comment:
```suggestion
const orderDetails = [
```
probably have to change below as well where you're mapping